apps/web/src/styles/components/auth.css

```diff
@@ -0,0 +1,235 @@
+.auth-grid{
+    @apply grid grid-rows-[100px_minmax(150px,_1fr)_350px] grid-flow-col;
+    @apply auto-rows-max;
+}
+
+.auth-title {
+    text-align: center;
+    font-size: 40px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+
+.auth-input-grid {
```

Comment:
do this in `tailwind` way. usage of tailwind in css file is good if you want to compose component style